syncto/views/errors.py

```diff
@@ -18,11 +19,8 @@ def response_error(context, request):
                              context.response.reason,
                              context.response.text)
 
-    statsd = request.registry.statsd
-
-    if statsd:
-        statsd.count("syncclient.status_code.%s" %
-                     context.response.status_code)
+    statsd_count(request, "syncclient.status_code.%s" %
+                 context.response.status_code)
```

Comment:
When I look at the code just below, it makes me wonder: do we want to log a `syncclient.status_code` when response is 304 ? Shouldn't we make sure the original error comes from Sync precisely ?